src/Microsoft.Azure.WebJobs.Host/Executors/StorageAccountProviderExtensions.cs

```diff
@@ -82,16 +86,27 @@ private static void ValidateStorageAccount(IStorageAccount account, string conne
         }
 
         /// <summary>
-        /// Walk from the parameter up to the containing type, looking for a
-        /// <see cref="StorageAccountAttribute"/>. If found, return the account.
+        /// Determine whether the specified parameter declares an explicit storage
+        /// account to use, either by specifying a value for <see cref="StorageAttribute.Account"/> or
+        /// if <see cref="StorageAccountAttribute"/> has been applied.
         /// </summary>
         internal static string GetAccountOverrideOrNull(ParameterInfo parameter)
```

Comment:
>GetAccountOverrideOrNull(ParameterInfo parameter) [](start = 31, length = 49)

Can we get rid of the overload that takes a ParameterInfo? (the other PR removed this function completely). Part of the goal here is to move away form ParameterInfo and rely exclusively on the Attribute, since the attribute is 1:1 with Function.json.